src/Icons/Accessibility.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+interface Props {
+  width?: string | number
+  height?: string | number
+  [x: string]: unknown // for the rest property
+}
+
+const Accessibility: React.FunctionComponent<Props> = ({
+  width = '78',
+  height = '32',
+  ...rest
+}) => (
+  <svg width={width} height={height} viewBox="0 0 78 32" fill="none" {...rest}>
+    <rect x="1" y="1" width="76" height="30" rx="15" fill="white" stroke="#E1E1E1" />
+    <path
+      d="M28.4692 6H21.5308C15.7148 6 11 10.7148 11 16.5308C11 22.3467 15.7148 27.0615 21.5308 27.0615H28.4692C34.2852 27.0615 39 22.3467 39 16.5308C39 10.7148 34.2852 6 28.4692 6Z"
+      fill="#2D2926"
```

Comment:
should we add the fillcolor prop for #2D2926 and a secondary color prop for "white"?

Reply:
yes! good idea